cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,43 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import ArticlePageObject from '../support/pages/articlePageObject';
+
+const articlePage1 = new ArticlePageObject();
+
 describe('User', () => {
+  let user;
+  let article;
+  let secondUser;
+
   before(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
+    cy.task('generateArticle').then((generateArticle) => {
+      article = generateArticle;
+    });
+
+    cy.task('generateUser').then((generateUser) => {
+      secondUser = generateUser;
+    });
   });
 
-  it.skip('should be able to follow the another user', () => {
+  beforeEach(() => {
+    cy.task('db:clear');
```

Comment:
Clearing the database in the `beforeEach` hook might not be necessary for every test case. Consider whether this operation is needed for each test or if it can be moved to the `before` hook to optimize test performance.